slamd/discovery/processing/algorithms/experiment_preprocessor.py

```diff
@@ -0,0 +1,94 @@
+from slamd.common.error_handling import SequentialLearningException, ValueNotSupportedException, \
+    SlamdUnprocessableEntityException
+from slamd.discovery.processing.models.model_type import ModelType
+
+
+class ExperimentPreprocessor:
+    @classmethod
```

Comment:
There should be an empty line between class declaration and decorator

Reply:
Added whitespace